recipe/meta.yaml

```diff
@@ -52,7 +53,7 @@ test:
     - write-torch-dir-to-table --help  # [not win]
   requires:
     - pip
-    - pytorch-cpu  # [not win]  # remove tag when conda-forge does pytorch builds for windows
+    - pytorch
```

Comment:
I found that if you add back the [not win] tag to the pytorch requirement, the Windows CI builds [pass](https://github.com/conda-forge/pydrobert-kaldi-feedstock/runs/33759563424). Would you mind adding it?

Reply:
Oh, sorry. I've wrongly assumed that the newer versions actually do include Windows builds, since the build files for Windows are there — but I guess they aren't actually being done.